Apps/W1/HybridSL/app/app.json

```diff
@@ -0,0 +1,49 @@
+{
+  "id": "3e3fe2fd-adb3-3417-5897-f35910a39759",
+  "name": "Dynamics SL Migration",
+  "publisher": "Microsoft",
+  "version": "25.0.0.0",
```

Comment:
Should be version 26. All references too.

Reply:
These values have been updated to 26.0.0.0 with Nikola's updates that he asked me to apply to this PR. These updates are included in commit ac98fd24f77480b2765b38152b3749913801c9b8